Vungle/VungleRouter.m

```diff
@@ -657,21 +652,30 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
         return;
     }
 
+    NSString *message = nil;
+    NSError *playabilityError = nil;
+    if (!isAdPlayable) {
+        message = error ? [NSString stringWithFormat:@"Vungle: Ad playability update returned error for Placement ID: %@, Error: %@", placementID, error.localizedDescription] : [NSString stringWithFormat:@"Vungle: Ad playability update returned Ad is not playable for Placement ID: %@.", placementID];
+        playabilityError = error ? : [NSError errorWithCode:MOPUBErrorAdapterFailedToLoadAd localizedDescription:message];
+    }
+
+    @synchronized (self) {
+        if (self.sdkInitializeState == SDKInitializeStateNotInitialized && !isAdPlayable) {
+            if ([self.waitingListDict objectForKey:placementID]) {
+                id<VungleRouterDelegate> delegate = [self.waitingListDict objectForKey:placementID];
+                [delegate vungleAdDidFailToLoad:playabilityError];
```

Comment:
@YueVungle 
I don't think we want to send "fail callback" at this point? Since SDK is not initialized yet, and we added the Placement ID in "waiting list", so we will request an ad for the placement once SDK initialized.
If we send "fail callback" at this point, we will lose one impression to fill in?

@crbinsf 
I think this root cause is that SDK send "playabilityUpdate" before SDK gets initialized. Those "playability" callbacks before init done, might be redundant.

Reply:
@ashinagawa Hi Aki, I agree with you. I will remove above block `@synchronized (self) {}`. We still need the delegate in the "waiting list" and will request an ad for the placement once SDK initialized.  Thanks a lot for the guidance. ^_^

@crbinsf Yes, Those "playability" callbacks before init done, might be redundant. We could remove them in our future releases. ^_^

Reply:
@ashinagawa Updated, thanks a lot. :)